README.rst

```diff
@@ -99,6 +99,17 @@ You can also specify the re-run delay time in the marker:
       import random
       assert random.choice([True, False])
 
+You can also specify an optional ``condition`` in the re-run marker:
+
+.. code-block:: python
+
+   @pytest.mark.flaky(reruns=5, condition=sys.platform.startswith("win32"))
```

Comment:
should the keyword be `on_condition` instead? 🤔 @sallner

Reply:
I am fine with `condition` as it is usually used as an enable. Otherwise I would expect something like `omit` or `deactivate`.